lib/Botkit.d.ts

```diff
@@ -0,0 +1,497 @@
+    save: (data: O, cb?: (err: Error, id: string) => void) => void;
+    get: (id: string, cb: (err: Error, data: O) => void) => void;
+    delete?: (id: string, cb?: (err: Error) => void) => void;
+    all?: (cb: (err: Error, data: O[]) => void) => void;
+  }
+  interface Studio<S, M extends Message, B extends Bot<S, M>> {
+    after(command_name: string, func: (convo: Conversation<M>, next: () => void) => void): this;
+    before(command_name: string, func: (convo: Conversation<M>, next: () => void) => void): this;
+    beforeThread(command_name: string, thread_name: string, func: (convo: Conversation<M>, next: () => void) => void): this;
+    get(bot: B, input_text: string, user: string, channel: string): Promise<Conversation<M>>;
+    run(bot: B, input_text: string, user: string, channel: string): Promise<Conversation<M>>;
+    runTrigger(bot: B, input_text: string, user: string, channel: string): Promise<Conversation<M>>;
+    validate(command_name: string, key: string, func: (convo: Conversation<M>, next: () => void) => void): this;
+  }
+  interface Team {
+    id: string;
+  }
+  interface TwilioIPMBot extends Bot<TwilioIPMSpawnConfiguration, TwilioIPMMessage> {
+    readonly api: any;
+  }
+  interface TwilioIPMConfiguration extends Configuration {
+  }
+  interface TwilioIPMController extends Controller<TwilioIPMSpawnConfiguration, TwilioIPMMessage, TwilioIPMBot> {
+    createWebhookEndpoints(webserver: any, bot: TwilioIPMBot): this;
+  }
+  interface TwilioIPMMessage extends Message {
+  }
+  interface TwilioIPMSpawnConfiguration {
+    autojoin?: boolean;
+    identity?: string;
+    TWILIO_IPM_SERVICE_SID: string;
+    TWILIO_ACCOUNT_SID: string;
+    TWILIO_API_KEY: string;
+    TWILIO_API_SECRET: string;
+  }
+  interface TwilioSMSBot extends Bot<TwilioSMSSpawnConfiguration, TwilioSMSMessage> {
+  }
+  interface TwilioSMSConfiguration extends Configuration {
+    account_sid: string;
+    auth_token: string;
+    twilio_number: string;
+  }
+  interface TwilioSMSController extends Controller<TwilioSMSSpawnConfiguration, TwilioSMSMessage, TwilioSMSBot> {
+    createWebhookEndpoints(webserver: any, bot: TwilioSMSBot, cb?: () => void): this;
+  }
+  interface TwilioSMSMessage extends Message {
+  }
+  interface TwilioSMSSpawnConfiguration {
+  }
+  interface User {
+    id: string;
+    name?: string;
+  }
+  type ConversationCallback<M extends Message> = ((message: M, convo: Conversation<M>) => void) | ({ pattern?: string | RegExp; default?: boolean; callback: (message: M, convo: Conversation<M>) => void; }[]);
+  type ConversationStatusType = 'completed' | 'active' | 'stopped' | 'timeout' | 'ending' | 'inactive';
+  type HearsCallback<S, M extends Message, B extends Bot<S, M>> = (bot: B, message: M) => void;
+  type HearsFunction<M extends Message> = (tests: string | string[] | RegExp | RegExp[], message: M) => boolean;
+  type SlackWebAPIMethod = (data: any, cb: (err: Error, response: any) => void) => void;
+}
+
+export = botkit;
```

Comment:
@Naktibalda Thank you for your review.
There is my mistake.
https://github.com/howdyai/botkit/pull/953/commits/5951c740ed0448c86f1d62643b69472a785767d2
Please retry by importing as below:
```typescript
import Botkit = require('botkit');
```